### ISSUE TITLE
fatal error when launching live replay

### ISSUE THREAD

**Issue by CookieNoob** [FAForever/fa#1654]
starting a live replay (this particular one, not all) via the crossed swords results in a fatal error for me and speed2. 
[crashwindow.txt](https://github.com/FAForever/fa/files/564009/crashwindow.txt)
[forever.txt](https://github.com/FAForever/fa/files/564010/forever.txt)
[game.txt](https://github.com/FAForever/fa/files/564011/game.txt)

**Comment by micheljung** [comment]
`info: >> CookieNoob is missing map /maps/setons_clutch_ultimate.v0003/setons_clutch_ultimate_scenario.lua`

**Comment by CookieNoob** [comment]
that is definitely not the problem.

I made setons_clutch_ultimate.v0003 and developed it into a newer version (v0004). I then downloaded v3 again, tested something and afterwards deleted it again. Since I didn't host a game after that, the lobby says I am missing the map, since it would be the map which is selected by default.

The map this game was played on was not setons. This bug also affected speed2, so I dont think the "missing setons" lines would have appeared in his log file...
